nav2_lifecycle_manager/include/nav2_lifecycle_manager/lifecycle_manager_client.hpp

```diff
@@ -29,30 +29,80 @@
 
 namespace nav2_lifecycle_manager
 {
-
+/**
+ * @enum nav2_lifecycle_manager::SystemStatus
+ * @brief Enum class representing the status of the system.
+ */
 enum class SystemStatus {ACTIVE, INACTIVE, TIMEOUT};
-
+/**
+ * @class nav2_lifecycle_manager::LifeCycleMangerClient
+ * @brief Implements lifecycle manager service client and NavgationToPose
+ * action client.
+ */
 class LifecycleManagerClient
 {
 public:
+  /**
+   * @brief A constructor for LifeCycleMangerClient
+   */
   LifecycleManagerClient();
 
   // Client-side interface to the Nav2 lifecycle manager
+  /**
+   * @brief Makes start up service call
```

Comment:
I would align same grammar in this file as you used in the `lifecycle_manager.hpp` (e.g. "**Make** start up service call").